boards/arm/mimxrt1020_evk/Kconfig.defconfig

```diff
@@ -15,13 +15,17 @@ endchoice
 config DISK_DRIVER_SDMMC
 	default y if DISK_DRIVERS
 
+if FLASH
+
 config FLASH_MCUX_FLEXSPI_NOR
-	default y if FLASH && BOARD_MIMXRT1020_EVK
+	default y
 
 config FLASH_MCUX_FLEXSPI_XIP
-	default y if FLASH && BOARD_MIMXRT1020_EVK
+	default y if (CODE_FLEXSPI || CODE_FLEXSPI2)
```

Comment:
`FLASH_MCUX_FLEXSPI_XIP` Kconfig already has this line defined,
`depends on (CODE_FLEXSPI || CODE_FLEXSPI2 || SOC_SERIES_IMX_RT6XX)`
Do we need ` if (CODE_FLEXSPI || CODE_FLEXSPI2)` here